### DISCUSSION TITLE
Question about access to Secret Projects

### DISCUSSION THREAD

**Discussion by JaggedOne** [induktio/thinker#63] [Q&A] [answered]
Now that I'm playing Thinker (which is awesome btw), I often see another faction start a Secret Project that I currently do not have availability to. I don't remember seeing this with the GOG release before I installed Thinker. Is this because (a) before Thinker, I always had access to a Secret Project started by another faction, or (b) in stock SMAC (w/o Thinker) the AI was dumb as a stump and rarely started Secret Projects ?

I think it is (b), but just wanted to check, as this is very noticeable with Thinker.

**Comment by induktio** [comment, marked as answer]
In the original game, AI will not start secret projects if the player does not have the prerequisite tech for them on the 3 lowest difficulty levels (Talent and below). This actually differs in Thinker AI, because it will skip this precondition and only consider if they themselves have the tech. Maybe this should be separately mentioned in the details. Previously this had its own config option but the wording was a little confusing so it was removed. For the original game, this seemed such an artificial constraint for the AI, it's probably not even necessary on the lowest difficulty levels since the AI production is already slower. At least that's what I would think. There are a couple of other obscure mechanics on different difficulty levels too, not really good design since they are hardcoded in the engine unless patched. They should be mentioned in some article.

**Comment by JaggedOne** [comment]
I always play on Specialist level because I like to be able to change my
builds w/o incurring penalties (I play for fun, not to beat any time or
meet any standards...lol).

Thanks for clearing that up, and thanks for your quick replies to my
questions !!!

BTW, Thinker is really great, and so much different than the original. The
other day I played a game and the Drones whooped up on me so bad I had to
just give up - first time that has ever happened. I usually stay isolated
until I am built up enough to destroy all the other factions. This time,
the Drones had covered half the planet by that time, and I lost about 75%
of the fights. For comparison, I went back to the vanilla game and played,
and the AI was so dumb I quit because it wasn't fun anymore. So I'm a
Thinker convert for sure !!!

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
